### ISSUE TITLE
how to pull the corresponding images when using non-x86 platform?

### ISSUE THREAD

**Issue by yunfan** [indigo-dc/udocker#353]
hi i had done a quick search in the sourcecode and i havnt found that `pull` command will try to pull the right image according to local architecture
and i had found that even on my arm cellphone (android 10 with termux), udocker pull the x86_64 relavent images of `alpine:latest`

if you were using docker command, you could just use `--platform` parameter, but how can we do that in udocker?

**Comment by mariojmdavid** [comment]
check updated user manual documentation in devel3 branch, soon to be released as v1.3.3

## 11. Host environment specific notes

### 11.1. Termux

udocker can be used with Termux on Android, the only mode currently supported
is **P** using PRoot. It is recommended to install and use the proot binary
provided by Termux which is adapted to the Termux Android environment.

```bash
export UDOCKER_USE_PROOT_EXECUTABLE=$(which proot)
udocker run arm64v8/fedora:35
```

**Comment by yunfan** [comment]
thank you very much, this release will open a brighting future for termux and udocker both
